core/controllers/topic_editor.py

```diff
@@ -224,6 +227,37 @@ def delete(self, topic_id):
         topic_services.delete_topic(self.user_id, topic_id)
 
 
+class TopicRightsHandler(base.BaseHandler):
+    """A handler for returning topic rights."""
+
+    @acl_decorators.can_manage_rights_for_topic
+    def get(self, topic_id):
+        """Returns the TopicRights object of a topic.
+        """
```

Comment:
Move to the end of the previous line.

/cc @aks681 @kevinlee12

Reply:
Done.